app/components/more-about.hbs

```diff
@@ -1,40 +1,43 @@
-<section class='info-video'>
-  <section class='info__section'>
-    <span data-test-note class='info__note'>
-      To know how this page is getting developed, join our Discord Channel.
-      Contact one of our existing
-      <a
-        data-test-members-link
-        class='info__note__link'
-        href={{this.MEMBERS_URL}}
-      >
-        members
-      </a>
-      for the invitation link or check our
-      <a data-test-faq-link class='info__note__link' href={{this.FAQ_URL}}>
-        FAQ
-      </a>
-      section.
-    </span>
-    <span class='info__img'>
-      <img
-        data-test-note-image
-        class='info__img-market'
-        src='assets/images/mobile-marketing.png'
-        alt='infoNoteImg'
-      />
-    </span>
-  </section>
-  <section class='video__section'>
-    <h1 data-test-video-title class='video__heading'>
+<section class="info-video">
+  {{! TODO - remove this after migration }}
+  {{#unless @isDev}}
```

Comment:
NIT: Satyam I Didn't understand this part
Why you put on under dev flag, If it is already

Reply:
all the new changes are in dev=true so that's why did this